src/components/dashboard/Dashboard.js

```diff
@@ -85,6 +87,27 @@ const Dashboard = props => {
     }
   }
 
+  //Service redirects Send/Receive
+  useEffect(() => {
+    const anyParams = extractQueryParams(window.location.href)
+    if (anyParams.code) {
+      const { screenProps } = props
+      if (anyParams && anyParams.code) {
```

Comment:
@StanislavShevchenko why there are two same conditions used in 95 and 93 lines?
The same for104 and 105 lines.

Reply:
Done